File: Controller/SaveController.cs
using Isocline.Model;
using Isocline.View;
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Isocline.Controller
{
    internal class SaveController
    {
        private SaveModel saveModel;
        public SaveController(SaveModel saveModel)
        {
            this.saveModel = saveModel;
        }
        public void SaveImage(string filePath)
        {
            saveModel.Save(filePath);
        }
    }
}

File: Controller/SaveController.vb
Imports Isocline.Model

Namespace Isocline.Controller
    Friend Class SaveController
        Private saveModel As SaveModel
        Public Sub New(ByVal saveModel As SaveModel)
            Me.saveModel = saveModel
        End Sub
        Public Sub SaveImage(ByVal filePath As String)
            saveModel.Save(filePath)
        End Sub
    End Class
End Namespace
